includes/Server.hpp

```diff
@@ -47,27 +52,31 @@ typedef std::map<int, Client> clientsMap;
 typedef std::map<std::string, Channel> channelsMap;
 typedef std::vector<std::string> stringVector;
 typedef std::pair<std::vector<std::string>, std::vector<std::string> >
-    pairOfStringVectors;
+    KeyValuePairList;
```

Comment:
```cpp
typedef std::pair<std::vector<std::string>, std::vector<std::string> > KeyValuePairList;
```

I prefer call it `KeyValuePairVector` because it's `std::vector`